### ISSUE TITLE
Add Text element defaults

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-android#1862]
First text element in a project should include:
 - Copy - Random translation of "Hello" `nice to have`
 - Font-Size : 20px (UI removed until we have multi-line)
 - Font Family : First option in the font dropdown (hide UI. depends on #1750) 
 - ~~Text Alignment : Center~~ (hide UI until we have multi-line)
 - Text Color : #E06A2C (Red option in the color bar)

Re-use the last used style for each new element in a project. A huge time saver for users. 

Remove `none` as a user color option unless there's a good use case. 

cc @flukeout

**Comment by thisandagain** [comment]
I can take this, though I'll wait until the end of the sprint as @k88hudson is making some large changes within these react components right now.

**Comment by xmatthewx** [comment]
As discussed with @flukeout - we should disable alignment and font size until we're ready to implement multi-line text.

**Comment by thisandagain** [comment]
:+1:
